tensorflow_serving/core/aspired_versions_manager.h

```diff
@@ -110,13 +110,13 @@ class AspiredVersionsManager : public Manager,
     ///
     /// If set as 0, we don't use a thread-pool, and servable loads are
     /// performed serially in the manager's main work loop.
-    uint32 num_load_threads = 0;
+    uint32 num_load_threads = 1;
 
```

Comment:
please do not change default values. instead add two new flags to the server (`num_load_threads` and `num_unload_threads`), so people can (optionally) tweak the settings as needed for their use case.

as an example, copy/follow the pattern on how `--max_num_load_retries` flag is implemented/used to set appropriate lower level options.

https://github.com/tensorflow/serving/blob/0acaa63b45a5c461dcb4808f8745ff9088a0f6e6/tensorflow_serving/model_servers/main.cc#L108

you can add new flags around this.

Reply:
ok, I had added two new flags to the server (num_load_threads and num_unload_threads).